front-end/packages/help/getting-started.md

```diff
@@ -1,26 +1,114 @@
+Sidebars provide instruments to manage your data and control text generation.
 
-Vivamus ipsum eros, congue id diam molestie, fringilla eleifend elit. Etiam sit amet quam dui. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Sed facilisis maximus enim. Vestibulum sodales feugiat consectetur. Fusce pulvinar diam sed augue accumsan volutpat. Sed aliquam nisl id pretium hendrerit. Curabitur aliquet sapien rhoncus finibus faucibus.
+![sidebars](help/sidebars.png)
 
-## One
+### Data
 
-Morbi sed velit non ipsum tempor maximus. Nulla elementum elementum magna, cursus rutrum lorem blandit in. Mauris et massa molestie, fermentum lectus mattis, sodales orci. Phasellus condimentum, purus vel aliquam facilisis, leo velit fermentum dui, eget ultricies risus mauris sit amet leo. Donec hendrerit porta turpis, ut fermentum odio ultrices laoreet. Donec volutpat ultricies sapien, at auctor ligula molestie quis. Maecenas lobortis augue at congue finibus. Aenean malesuada quis orci ac ornare. Donec efficitur dolor vitae mauris auctor cursus. Morbi sed ante sit amet urna vehicula scelerisque quis a erat. Integer imperdiet ipsum at accumsan bibendum. Nunc ultrices non tellus eget aliquam. Duis a sapien vel ligula vulputate lacinia eu eleifend purus. Curabitur vehicula euismod sem quis eleifend.
+*Data* sidebar provides the tools to load CSV data files to be used in generation. Note that CSV files have to be comma separated with the column headers at the top.
 
-Vestibulum vitae odio vel odio gravida tincidunt. Phasellus vehicula, ex sed vehicula semper, arcu turpis tincidunt lacus, quis egestas augue lectus non tellus. Maecenas est nunc, semper eget tellus nec, congue dictum libero. Nunc sit amet tellus maximus, mattis est eget, scelerisque felis. Mauris dapibus eros nec ipsum venenatis facilisis. Maecenas interdum auctor neque, accumsan dapibus erat. Vivamus vitae ullamcorper quam. Nulla blandit sapien id eleifend blandit. Curabitur id volutpat enim. Nunc tellus mauris, blandit nec fringilla vitae, sollicitudin in ipsum. Cras viverra sapien felis, et lobortis diam luctus at. Quisque et dui id turpis finibus accumsan sed eu dolor.
+More than one data file can be loaded with the help of *Add* button. File selected in the drop down menu is a current one and will be used for the generation.
```

Comment:
"is a current" -> "is the current", "the generation" -> "generation" or "text generation"